app/views/invitations/new.html.haml

```diff
@@ -6,27 +6,28 @@
   .control-group
     = invite_link(@invite_code)
 
-#email_invitation
-  = form_tag new_user_invitation_path, class: 'form-horizontal' do
+- if AppConfig.mail.enable?
+  #email_invitation
+    = form_tag new_user_invitation_path, class: 'form-horizontal' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.